### ISSUE TITLE
Style selected item of the sorting dropdown

### ISSUE THREAD

**Issue by let4be** [lavab/web#249]
![image](https://cloud.githubusercontent.com/assets/4147061/6592249/4925d63a-c7dc-11e4-8e1f-6f09d776a20b.png)

**Comment by piggyslasher** [comment]
@vLooz - assigning to you because I have no idea how to proceed with this. the functionality is there but no sorting GUI.

**Comment by vLooz** [comment]
I'd leave this out for now @piggyslasher. In the future it could be nice to give user the choice between having the newest mails at the top or the bottom. But not MVP from my POV. What do you think @andreis ?

**Comment by piggyslasher** [comment]
cool. @flixi - can we have a new milestone - say `2.2`? I can't seem to find the place to add it. Maybe I don'e have the privilege.

**Comment by andreis** [comment]
Yeah,  sure - newest top/bottom as a Settings choice (post-MVP), bottom default.

cc @pzduniak @vLooz

**Comment by vLooz** [comment]
I'd prefer top on default :-)

**Comment by flixi** [comment]
This is definitely a 2.1 issue!
We have yet to get 2.1 completely defined. Sorry, so long as we haven'T defined a comprehensive roadmap for this, all isues which are not 2.0 are 2.1...

**Comment by andreis** [comment]
> Tine, an alien race which is the focus of the award-winning novel A Fire Upon the Deep and its sequel The Children of the Sky

??
